From be3acf296a230b13cf27409d6f313c810277897e Mon Sep 17 00:00:00 2001
From: sylvain <sylv1pdt@gmail.com>
Date: Fri, 12 Mar 2021 22:34:44 +0100
Subject: [PATCH] Force Gtk version 3.0

Import fails when using gtk4: AttributeError: 'gi.repository.Gtk'
object has no attribute 'Container'
https://gitlab.gnome.org/GNOME/pygobject/-/issues/423
---
 playitslowly/app.py | 1 +
 1 file changed, 1 insertion(+)

diff --git a/playitslowly/app.py b/playitslowly/app.py
index 32ad43b..9dbcbc2 100755
--- a/playitslowly/app.py
+++ b/playitslowly/app.py
@@ -32,6 +32,7 @@
 
 import gi
 gi.require_version('Gst', '1.0')
+gi.require_version('Gtk', '3.0')
 
 from gi.repository import Gtk, GObject, Gst, Gio, Gdk
 
